### PR TITLE
Fix docker image migration

### PR DESCRIPTION
**Pull request by andriisoldatenko**
fix https://github.com/astronomer/issues/issues/566
added small refactoring.

### CODE REVIEW

**Comment by schnie** [comment]
So which part of this was I hitting when upgrading? `pull` vs `push`?

File: src/migrations/20191101143903_migrate_initial_docker_image.js [hunk 1]
```diff
@@ -42,7 +45,7 @@ export async function up() {
           type: "repository",
           // Build the repo name.
           name: repo,
-          actions: ["push"]
+          actions: ["pull"]
```
Comment (andriisoldatenko) [inline src/migrations/20191101143903_migrate_initial_docker_image.js]:
```
JWT__REGISTRY__CERT_PATH=. JWT__REGISTRY__SERVICE=docker-registry JWT__REGISTRY__ISSUER=houston NODE_ENV=development PUBLIC_SIGNUPS=true npx babel-node node_modules/.bin/knex "migrate:latest"
Using environment: development
2019-11-18T17:05:04 DEBUG Running prisma-deploy
Deploying service `houston` to stage `default` to server `local` 547ms
Service is already up to date.
2019-11-18T17:05:06 DEBUG Migrating advanced-quark-6540...
2019-11-18T17:05:06 DEBUG Requesting docker tags for advanced-quark-6540 at http://localhost:5001/v2/advanced-quark-6540/airflow/tags/list
```
@schnie ^ this line

File: src/migrations/20191101143903_migrate_initial_docker_image.js [hunk 2]
```diff
@@ -57,9 +60,11 @@ export async function up() {
         headers: { Authorization: `Bearer ${dockerJWT}` }
       });
 
-      let tag;
-
-      for (tag of tags) {
+      if (tags.length === 0) {
```
Comment (andriisoldatenko) [inline src/migrations/20191101143903_migrate_initial_docker_image.js]: this line we need to fix: `2019-11-18T16:54:46 ERROR tags is not iterable`

File: src/migrations/20191101143903_migrate_initial_docker_image.js [hunk 3]
```diff
@@ -57,9 +60,11 @@ export async function up() {
         headers: { Authorization: `Bearer ${dockerJWT}` }
       });
 
-      let tag;
-
-      for (tag of tags) {
+      if (!tags) {
+        log.info("There is no tags :(");
+        continue;
+      }
```
Comment (schnie) [inline src/migrations/20191101143903_migrate_initial_docker_image.js]:
```suggestion
      }

```

**Pull request merged**